tests/pytests/test_cassini_drivers.py

```diff
@@ -61,8 +61,7 @@ def test_kernels(scope="module", autouse=True):
 def test_cassini_load(test_kernels, usgscsm_compare_dict):
     label_file = get_image_label("N1702360370_1")
     usgscsm_isd = ale.load(label_file, props={'kernels': test_kernels}, formatter='usgscsm')
-    print(usgscsm_isd['sensor_position'])
-    print(usgscsm_isd['sensor_orientation'])
+    print(usgscsm_isd)
```

Comment:
Left over print

Reply:
I actually left this in because it is nice to see the entire ISD when you get errors.